packages/block-library/src/post-featured-image/editor.scss

```diff
@@ -1,33 +1,136 @@
-div[data-type="core/post-featured-image"] {
-	img {
-		max-width: 100%;
-		height: auto;
-		display: block;
+// Give the featured image placeholder the appearance of a literal image placeholder.
+// @todo: this CSS is similar to that of the Site Logo. That makes it an opportunity
```

Comment:
Most of this code is copied verbatim from #35397, and suggests there is an opportunity to create a new component, such as `InheritingPlaceholder` (it'd need a better name), which comes with less content inside, usually just a single upload/media library button, minimal markup, and intentionally lets theme styles bleed in and affect it.

It doesn't necessarily have to happen as part of this PR, but it seems worth tracking if this PR goes any further.